### PR TITLE
use eslint-config-exp, removed skipped testcase and audit fix

### PR DESCRIPTION
**Pull request by markusn**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,31 +52,31 @@ function buildFetch(behavior) {
   }
   const requestTimeFn = behavior.requestTimeFn || defaultRequestTimeFn;
 
-  if (behavior.hasOwnProperty("clone")) {
+  if (Object.prototype.hasOwnProperty.call(behavior, "clone")) {
```
Comment (paed01) [inline index.js:55]: Not sure why this rule is necessary in a backend library? Or we could use `"clone" in behaviour`.

File: test/lib/cloningTest.js [hunk 2]
```diff
@@ -12,30 +12,30 @@ describe("Fetching redirected resources", () => {
   function fakeResponse(pathName) {
     fake
       .get(pathName)
-      .reply(200, {a: 1, b: 2});
+      .reply(200, { a: 1, b: 2 });
```
Comment (paed01) [inline test/lib/cloningTest.js:15]: another unnecessary rule IMO.

File: index.js (hunk 1 above)
Comment (markusn) [reply index.js:55]: In this particular case the rule isn't necessary, but is there a reason to not use Object.prototype.hasOwnProperty.call ?

Comment (paed01) [reply index.js:55]: there is no reason if you are a developing a backend project. Though we could argue that this rule will nudge devs to use in operator.

File: test/lib/cloningTest.js (hunk 2 above)
Comment (markusn) [reply test/lib/cloningTest.js:15]: its' part of the standard to ensure that the same spacing is used everywhere

File: index.js (hunk 1 above)
Comment (markusn) [reply index.js:55]: The rule applies to any project where the input may come from a client, including backend projects. However in this particular case it is not necessary but on the other-hand it doesn't hurt to use it either.

Write

**Review by paed01 — APPROVED** [review]

**Pull request merged**